README.md

```diff
@@ -4,25 +4,22 @@ Thanks to everyone that is starring, forking, writing issues, pull requesting an
 
 ![updatedgif45729805790](https://user-images.githubusercontent.com/81049050/150833862-0a828939-f267-4bd2-931f-79df55d51e28.gif)
 
-# Dependecies (all Python dependecies will automatically be installed):
-1.  mpv player https://mpv.io/  (Must be installed through a package manager)
-2.  youtube-dl https://github.com/ytdl-org  (Python dependency)
+## Dependecies:
+- mpv player https://mpv.io/  (Must be installed through a package manager)
 
-------
```

Comment:
Not a fan of these since the `#` in markdown already creates a line to separate sections when rendered on github